scripts/fuzz_opt.py

```diff
@@ -275,12 +275,28 @@ def compare(x, y, context):
         ))
 
 
+# converts a possibly-signed integer to an unsigned integer
+def unsign(x, bits):
+    return x & (2**bits - 1)
```

Comment:
I think better use:
```suggestion
    return x & ((1 << bits) - 1)
```
Python can't optimize pretty expensive power